docs/config/setup.adoc

```diff
@@ -2,7 +2,7 @@
 :icons: font
 :sectnumlevels: 2
 :imagesdir: images
-:chapter-label: 
+:chapter-label:
```

Comment:
This space was a feature—and still is! Please add it again. It changes `Chapter X` to `X`